docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,241 @@
+.. _windows_sources:
+
+Windows installation from sources
```

Comment:
A lot of my previous comments apply to this file

Reply:
ok.